data/categories.txt

```diff
@@ -1263,7 +1263,7 @@ nl:^Sportartikelen
 fi:^Urheilukauppa
 fr:^Articles de sport
 de:^Sportartikel|Fitnessgeschäft|2Laden
-hu:^Sporteszközök
+hu:^Sporteszközbolt
```

Comment:
From what I understand, you're replacing things like "sports equipment" to "sports equipment shop" in several places. Are you sure that people do not search for these shops by typing the terms of the former kind?
That is, if you think it is common to type "Sporteszközök" when searching for this shop, please leave the synonym in here and in other places: ```hu:^Sporteszközbolt|Sporteszközök```.
The "^" sign is before the term that will be shown to the user, the other synonyms are there only to help the search.

Reply:
OK, it is not a big difference so may have both or just have "sporteszközök". I added "bolt" because it is a shop so it is perhaps more exact that way.​